src/collections/Form/Form.js

```diff
@@ -7,8 +8,22 @@ export default class Form extends Component {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
+    settings: PropTypes.object,
   };
 
+  componentDidMount() {
+    this.element = $(this.refs.element);
+    this.element.form(this.props.settings);
+  }
+
+  componentWillUnmount() {
+    this.element.off();
+  }
+
+  plugin() {
+    return this.element.form(...arguments);
+  }
+
```

Comment:
[No portals necessary](https://github.com/ryanflorence/react-training/blob/gh-pages/lessons/05-wrapping-dom-libs.md#portals)! This is by far the simplest approach I could've imagined. I ran some memory profiling and everything looks tip top from the browser GC perspective over multiple form interactions and submissions:

<img width="540" alt="screen shot 2015-12-15 at 9 11 20 am" src="https://cloud.githubusercontent.com/assets/440298/11830768/9f434d14-a36d-11e5-968f-bc8245b846e5.png">